modelling/src/main/java/org/axonframework/modelling/saga/PayloadAssociationResolver.java

```diff
@@ -62,17 +62,27 @@
     }
 
     private <T> Property createProperty(String associationPropertyName, MessageHandlingMember<T> handler) {
+        if (associationPropertyName.isEmpty()) {
```

Comment:
Curious once more: why is this added in the multi-tenancy support (draft) PR?

Reply:
not sure how it ended up in the PR, this change is quite old and from master branch